Lib/test/test_gettext.py

```diff
@@ -717,6 +717,84 @@ def test_expand_lang(self):
                                          return_value=locale):
                     self.assertEqual(gettext._expand_lang(locale), expanded)
 
+# helper for FindTestCase
+def _clearvars(env):
+    for key in ('LANGUAGE', 'LC_ALL', 'LC_MESSAGES', 'LANG'):
+        env.unset(key)
+
+class FindTestCase(unittest.TestCase):
+    def test_find_with_env_vars(self):
+        # test that find correctly finds the environment variable LANGUAGE
+        # when languages are not supplied
+        with os_helper.EnvironmentVarGuard() as env, os_helper.temp_cwd() as tempdir:
```

Comment:
Looks like this setup is needed in every method here; you could deduplicate with something like:
```
def setUp(self):
    self.env = self.enterContext(os_helper.EnvironmentVarGuard())
    self.tempdir = self.enterContext(os_helper.temp_cwd())
```